doc/changes/changes_0.5.0.md

```diff
@@ -26,4 +26,6 @@ This enables faster and more robust tests for the pure CLI-related features, fas
 
 ## Bug Fixing
 
-* #60: Fix handling pip requirements when creating an SLC
+* #60: Fixed handling pip requirements when creating an SLC.
+* #58: Fixed the bug in language_container_builder.find_path_backwards.
```

Comment:
what about  https://github.com/exasol/python-extension-common/issues/35
and https://github.com/exasol/python-extension-common/issues/36 ?

Reply:
Very good point. I forgot to finish the change list. Thanks.